oak/server/oak_node.cc

```diff
@@ -92,6 +93,70 @@ static wabt::Result ReadModule(const std::string module_bytes, wabt::interp::Env
   return result;
 }
 
+struct RequiredExport {
+  std::string name_;
+  bool mandatory_;
+  wabt::interp::FuncSignature sig_;
+};
+
+const RequiredExport kRequiredExports[] = {
+    {"oak_initialize", true, {}},
+    {"oak_handle_grpc_call", true, {}},
+    {"oak_finalize", false, {}},
+};
+
+static bool CheckModuleExport(wabt::interp::Environment* env, wabt::interp::DefinedModule* module,
```

Comment:
Could you add a comment explaining what the return value means?

Reply:
Done.